tests/test_bdist_pex.py

```diff
@@ -104,6 +104,48 @@ def test_entry_points_ini_string():
     )
 
 
+def test_bdist_all_single_entry_point_dict():
+    # type: () -> None
+    assert {"first_app"} == set(
+        assert_entry_points({"console_scripts": ["first_app = my_app.my_module:do_something"]}, ["--bdist-all"]))
+
+
+def test_bdist_all_two_entry_points_dict():
+    # type: () -> None
+    assert {"first_app", "second_app"} == set(
+        assert_entry_points({"console_scripts": ["first_app = my_app.my_module:do_something",
+                                                 "second_app = my_app.my_module:do_something"]}, ["--bdist-all"]))
```

Comment:
You're definitely going to get formatting errors when CI runs. Please run `tox -eformat-run` to fix these up.

Reply:
Updated